integrations/vello_svg/src/lib.rs

```diff
@@ -197,20 +229,21 @@ fn paint_to_brush(paint: &usvg::Paint, opacity: usvg::Opacity) -> Option<(Brush,
                     cstop.color.g = stop.color.green;
                     cstop.color.b = stop.color.blue;
                     cstop.color.a = (stop.opacity * opacity).to_u8();
-                    cstop.offset = stop.offset.get() as f32;
+                    cstop.offset = stop.offset.get();
                     cstop
                 })
                 .collect();
-            let start: vello::kurbo::Point = (gr.x1, gr.y1).into();
-            let end: vello::kurbo::Point = (gr.x2, gr.y2).into();
-            let transform = Affine::new([
-                gr.transform.a,
-                gr.transform.b,
-                gr.transform.c,
-                gr.transform.d,
-                gr.transform.e,
-                gr.transform.f,
-            ]);
+            let start: vello::kurbo::Point = (gr.x1 as f64, gr.y1 as f64).into();
```

Comment:
I think I'd use `Point::new` or `Point::from` here

Reply:
Ah, I see you're copying the extant style

In that case, I guess either is fine

Reply:
I also think it should be a common convention across the file. I'll try to make it consistent when I get to my next revision.